.bazelrc

```diff
@@ -15,16 +15,15 @@ build:clang-cl --compiler=clang-cl
 build:msvc --compiler=msvc-cl
 # `LC_ALL` and `LANG` is needed for cpp worker tests, because they will call "ray start".
 # If we don't add them, python's `click` library will raise an error.
-test --action_env=LC_ALL
-test --action_env=LANG
+build --action_env=LC_ALL
+build --action_env=LANG
 # Allow C++ worker tests to execute "ray start" with the correct version of Python.
-test --action_env=VIRTUAL_ENV
-test --action_env=PYENV_VIRTUAL_ENV
-test --action_env=PYENV_VERSION
-test --action_env=PYENV_SHELL
-test --action_env=RAY_ENABLE_NEW_SCHEDULER
+build --action_env=VIRTUAL_ENV
+build --action_env=PYENV_VIRTUAL_ENV
+build --action_env=PYENV_VERSION
+build --action_env=PYENV_SHELL
```

Comment:
These could probably be removed; you'd typically find these kind of virtual environment configs in each developer's personal .bazelrc, given that virtual environment usage varies from dev to dev.